src/main/java/org/commcare/formplayer/objects/SerializableFormSession.java

```diff
@@ -42,6 +42,9 @@ public enum SubmitStatus {
     @Column(name = "appid", updatable = false)
     private String appId;
 
+    @Column(name = "appversion", updatable = false)
+    private String appVersion;
```

Comment:
Think it should not but think we should verify that it doesn't break Incomplete forms on resuming after deploy (can be part of QA)